scripts/update_flowcapture.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+MANIFEST_FILE="flow-capture.yml"
+MANIFEST_PATH="res"
+MANIFEST_OUTPUT_PATH="output"
+
+if ! command -v yq &> /dev/null
+then
+    echo "yq binary not found, installing... "
+    go install -mod='' github.com/mikefarah/yq/v4@v4.43.1
+fi
```

Comment:
:point_up: I would like to avoid as much as possible dependencies in the script

That will help to keep cross platform compatibility

Reply:
I removed it from the script and added to `make prereqs` target